Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -171,14 +171,11 @@ namespace Multiplayer
             if (m_serverProcessWatcher)
             {
                 m_serverProcessWatcher->TerminateProcess(0);
-                if (m_serverProcessTracePrinter)
-                {
-                    m_serverProcessTracePrinter->Pump();
-                    m_serverProcessTracePrinter->WriteCurrentString(true);
-                    m_serverProcessTracePrinter->WriteCurrentString(false);
-                }
-                m_serverProcessWatcher = nullptr;
+
+                // the TracePrinter hangs onto a pointer to an object that is owned by
```

Comment:
Minor: Other comments start with capital letter; might look better as "// The Trace..."